mvlearn/cluster/mv_spectral.py

```diff
@@ -79,6 +76,15 @@ class MultiviewSpectralClustering(BaseEstimator):
         Only used if nearest neighbors is selected for affinity. The
         number of neighbors to use for the nearest neighbors kernel.
 
+    Attributes
+    ----------
+    labels_ : array-like, shape (n_samples)
```

Comment:
can you add a comma to make this "(n_samples,)"

Reply:
Will do